components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/main/java/org/wso2/carbon/identity/functions/library/mgt/dao/impl/FunctionLibraryDAOImpl.java

```diff
@@ -105,10 +112,10 @@ public void createFunctionLibrary(FunctionLibrary functionLibrary, String tenant
      * @param functionLibraryName Function library name
      * @param tenantDomain        Tenant domain
      * @return Function library
-     * @throws FunctionLibraryManagementException
+     * @throws FunctionLibraryManagementServerException
      */
     public FunctionLibrary getFunctionLibrary(String functionLibraryName, String tenantDomain)
-            throws FunctionLibraryManagementException {
+            throws FunctionLibraryManagementServerException {
```

Comment:
why this change is required? isn't FunctionLibraryManagementServerException a subclass of FunctionLibraryManagementException? In that case, we can keep the method interface as it is

Reply:
please check other places as well

Reply:
[Fixed commit](https://github.com/wso2/carbon-identity-framework/pull/2679/commits/9a526929402c0b6fb5ef8eab5d87a89fbaeee7c8)